README.md

```diff
@@ -59,7 +59,6 @@ Some example requests are as follows:
 * `*/oai2?verb=Identify`
 * `*/oai2?verb=ListMetadataFormats`
 * `*/oai2?verb=ListIdentifiers&metadataPrefix=oai_dc`
-* `*/oai2?verb=GetRecord&metadataPrefix=oai_dc&identifier=PID`
```

Comment:
Why was the GetRecord verb removed from the documentation?

Reply:
That request returns the following error:
<error code="idDoesNotExist">
The value 'PID' of the identifier is illegal for this repository.
</error>

Reply:
You need the actual identifier for the object; "PID" is just a placeholder.

e.g. https://arcabc.ca/oai2?verb=GetRecord&metadataPrefix=oai_dc&identifier=islandora:root

Reply:
Embarrassing mistake on my part. Thanks for clarifying. I will add that request back into the documentation and submit a new PR.

Reply:
You don't need to create a new PR, just add the line back to this PR with another commit.

Reply:
Thanks. Done.

Reply:
<bump>